lib/md055.js

```diff
@@ -0,0 +1,27 @@
+// @ts-check
+
+"use strict";
+
+const { addErrorContext } = require("../helpers");
+const { filterByTypes } = require("../helpers/micromark.cjs");
+
+module.exports = {
+    "names": [ "MD055", "table-missing-border" ],
+    "description": "Table is missing leading or trailing pipe character",
+    "tags": [ "table" ],
+    "function": function MD055(params, onError) {
+        const tables = filterByTypes(params.parsers.micromark.tokens, [ "table" ]);
+        for (const table of tables) {
+            const rows = filterByTypes([ table ], [ "tableRow", "tableDelimiterRow" ]);
```

Comment:
I know I said to use table, but I should have said to use table.children - we know the table object won't match and table.children is already an array.